tests/unit/FetchErrorDialog.spec.ts

```diff
@@ -15,7 +14,7 @@ const store = useStore()
 // Prevent the warning "[Vuetify] Unable to locate target [data-app]"
 document.body.setAttribute('data-app', 'true')
 
-describe.skip('Fetch Error Dialog', () => {
+describe('Fetch Error Dialog', () => {
```

Comment:
These tests were being skipped.  I've re-enabled the tests and then changed the error message to match current behavior.  If you prefer, I can skip the test again.

Reply:
IMHO, if the tests pass, leave them. If they fail, fix them or skip them again.

Reply:
Thanks for the clarification.  I've enabled the tests and made sure they're passing.  In retrospect, I should have left the tests skipped, but now that they're passing I'd like to leave them as is.